client/src/FormFrontend/FormFrontend.js

```diff
@@ -297,6 +296,42 @@ class FormFrontend extends React.Component {
             </Fragment>
           );
         }
+        case "form_faq":
+          return (
+            <>
+              <div style={{ fontWeight: 700 }}>
+                {text_maker("form_frontend_faq_note")}
+                {required_asterisk}
+              </div>
+              <FancyUL>
+                {_.map(field_info.faq_content, ([q, a], idx) => {
+                  return (
+                    <div key={idx}>
+                      <div style={{ fontWeight: "bold" }}>{text_maker(q)}</div>
+                      <TM k={a} />
+                    </div>
+                  );
+                })}
+              </FancyUL>
+              <CheckBox
+                id={"form_frontend_faq"}
+                active={faq_acknowledged}
+                disabled={disable_forms}
+                onClick={() =>
+                  this.setState({
+                    faq_acknowledged: !faq_acknowledged,
+                    completed_template: {
+                      ...completed_template,
+                      [field_key]: true,
+                    },
+                  })
```

Comment:
As mentioned above, this should use `mergeIntoCompletedTemplateState`, like the other fields. Might need to toggle the value between `true` and `null` rather than `true` and `false`, so that the validation works as expected. Double check that.